src/test/java/com/google/api/codegen/transformer/go/testdata/myproto.proto

```diff
@@ -2,10 +2,14 @@ syntax = "proto3";
 
 package google.example.myproto.v1;
 
+import "singleservice.proto";
+
 service Gopher {
   rpc SimpleMethod(SimpleRequest) returns (SimpleResponse);
   rpc RetryMethod(SimpleRequest) returns (SimpleResponse);
   rpc PageStreamMethod(PageStreamRequest) returns (PageStreamResponse);
+
+  rpc MonkeyFood(SimpleRequest) returns (google.example.odd.v1.SpicyBanana);
```

Comment:
I think naming the method as a command would be more natural, e.g. "FeedMonkey".